perceval/backends/gerrit.py

```diff
@@ -258,24 +323,38 @@ def next_retrieve_group_item(self, last_item=None, entry=None):
                 next_item = 0
             else:
                 next_item = last_item
+        elif gerrit_version[0] == 2 and gerrit_version == 9:
+            # https://groups.google.com/forum/#!topic/repo-discuss/yQgRR5hlS3E
+            raise RuntimeExecption("Gerrit 2.9 does not support pagination")
         else:
             if entry is not None:
                 next_item = entry['sortKey']
 
         return next_item
 
-    def _get_gerrit_cmd(self, last_item):
+    def _get_gerrit_cmd(self, last_item, filter_=None):
+
+        if filter_ and filter_ not in ['status:open','status:closed']:
+            raise RuntimeError("Filter not supported in gerrit %s" % (filter_))
 
         cmd = self.gerrit_cmd + " query "
         if self.project:
             cmd += "project:"+self.project+" "
         cmd += "limit:" + str(self.max_reviews)
 
-        if self.blacklist_reviews:
-            blacklist_reviews = ' AND NOT (%s)' % (','.join(self.blacklist_reviews))
-            cmd += " '(status:open OR status:closed)%s' " % (blacklist_reviews)
+        if not filter_:
```

Comment:
I have already reword it trying to simplify it. Maybe we can improve it a little more but not sure. There are subtle differences between filter and not filter commands.